README.md

```
@@ -79,14 +101,16 @@ php artisan db:seed --class=ExampleRecipesSeeder
 
-| Environment Variable          | Description                                                        | Default           |
-|-------------------------------|--------------------------------------------------------------------|-------------------|
-| `TRMNL_PROXY_BASE_URL`        | Base URL of the native TRMNL service                               | https://trmnl.app |
-| `TRMNL_PROXY_REFRESH_MINUTES` | How often should the server fetch new images from native service   | 15                |
-| `REGISTRATION_ENABLED`        | Allow user registration via Webinterface                           | 1                 |
-| `FORCE_HTTPS`                 | If your server handles SSL termination, enforce HTTPS.             | 0                 |
-| `PHP_OPCACHE_ENABLE`          | Enable PHP Opcache                                                 | 0                 |
-| `TRMNL_IMAGE_URL_TIMEOUT`     | How long TRMNL waits for a response on the display endpoint. (sec) | 30                |
+| Environment Variable          | Description                                                                                                                                 | Default           |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| `TRMNL_PROXY_BASE_URL`        | Base URL of the native TRMNL service                                                                                                        | https://trmnl.app |
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)